app/lib/actions/pulp3/repository/commit_upload.rb

```diff
@@ -0,0 +1,22 @@
+module Actions
+  module Pulp3
+    module Repository
+      class CommitUpload < Pulp3::AbstractAsyncTask
+        def plan(repository, smart_proxy, upload_href, sha256)
+          plan_self(:repository_id => repository.id, :smart_proxy_id => smart_proxy.id, :upload_href => upload_href, :sha256 => sha256)
+        end
+
+        def invoke_external_task
+          if input[:skip]
+            output[:pulp_tasks] = nil
+          else
```

Comment:
could this just be re-written as  'unless input[:skip]' ?  i'm not sure setting output[:pulp_tasks] to nil is really worth anything?

Reply:
also i don't even see skip being passed into plan_self, so i'm not sure how it would ever be set?